### ISSUE TITLE
Mute doesn't work.

### ISSUE THREAD

**Issue by theDraco** [kav2k/silver_feather#31]
Hi, first I want to say a huge THANK YOU for adding the new enhancements for viewing images and quoted tweets. These two features make this extension 100 times better, no exaggeration. I've been waiting on that image feature forever! It's one of the reasons I switched over to Silver Feather from Silver Bird in the first place. Silver Feather had an image preview feature, but this new feature where you can see the entire image is just amazing! If you take donations, let me know and I'll donate something.

The reason for this post is to report that the "mute" feature doesn't work. For instance, if you put someone on "mute" their tweets will still show up in Silver Feather. Also, I believe the same is true if you block someone, their tweets will still show up in Silver Feather.

Could someone please address these issues?

P.S. One note on the quoted tweet feature. I noticed that if a tweet that's quoted is also quoting a tweet,   the quoted tweet from the quoted tweet will not show up. I hope that makes sense. For instance, if a tweet quotes a tweet, the quoted tweet shows up below it, which is great. But if that quoted tweet below the original tweet is also quoting another tweet, that quoted tweet doesn't show up. It would be cool if you it showed all quoted tweets in the entire chain of tweets.

**Comment by kav2k** [comment]
You should've posted 2 separate issues, but oh well.

The quote chain will be available today; however, chained quotes do not carry the information about the user quoted, so until you expand it at least once it will be "quoted (someone)". And quote chains automatically expand to at most one level - after that, you have to do it manually like with replies.

Mutes and blocks - I don't have a timeline for that. I'll look into it, but I'm not sure when.

**Comment by theDraco** [comment]
Hi, thanks for the reply. Yes, I see you can expand a quote within a quote now, that's great! One other thing I noticed with quoting tweets, if someone retweets a tweet that's quoting a tweet, the quoted tweet won't show up. It would be nice if you would see quoted tweets in a retweet.

For the mute and block features, those are really important, I hope you have a chance to address them.

This really is hands down the best Twitter extension now. IT'S AMAZING! Thanks again!

**Comment by kav2k** [comment]
Could you make a separate issue for that? It's hard to track otherwise.
